assets/js/blocks/checkout/billing/index.js

```diff
@@ -31,10 +36,10 @@ const getFieldBlock = ( field ) => {
 		case 'email':
 		case 'tel':
 		case 'text':
-			return [ 'woocommerce/checkout-input', { ...attributes, type: field.type } ];
+			return [ 'woocommerce/checkout-input', { ...attributes, ...{ type: field.type, hasSettings: withSettings.includes( field.autocomplete ) } } ];
```

Comment:
Rather than using two spread operators, you can add each attribute on like this:
```suggestion
			return [ 'woocommerce/checkout-input', { ...attributes, type: field.type, hasSettings: withSettings.includes( field.autocomplete ) } ];
```

Also, `field.autocomplete` is not an intuitive name, but I guess that comes from wc core so 🤷🏼‍♀️